<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task3</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="4">
     <widget class="QPushButton" name="m_buttonStart">
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>UserName:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="m_lineEditUN"/>
    </item>
    <item row="1" column="5">
     <widget class="QPushButton" name="m_buttonStop">
      <property name="text">
       <string>Стоп</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="m_lineEditP"/>
    </item>
    <item row="0" column="0" colspan="7">
     <widget class="QTableView" name="m_tableView"/>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="m_buttonTest1">
      <property name="text">
       <string>Тест1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="m_buttonTest2">
      <property name="text">
       <string>Тест2</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="m_buttonDeleteTable">
      <property name="text">
       <string>Удалить таблицу</string>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QPushButton" name="m_buttonCreateTable">
      <property name="text">
       <string>Создать таблицу</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
